.golangci.yml

```diff
@@ -9,7 +9,7 @@ issues:
   exclude-use-default: false
 
   exclude-rules:
-    - path: "(central|compliance|integration-tests|local|migrator|operator|pkg|sensor|tests|tools|scale)/"
+    - path: "(central|compliance|integration-tests|local|migrator|operator|pkg|roxctl|sensor|tests|tools|scale)/"
```

Comment:
Why do we need that? If we exclude it here then we can disable this linter

Reply:
@janisz, Thank you for pointing this out!

I thought that other parts were excluded because they use `CausedBy`. But then I have checked commit history, and actually, it was like that from the beginning. I have changed Regex because it was there for wrappers that we have replaced.

✅